repos/system_upgrade/el7toel8/models/opensshconfig.py

```diff
@@ -0,0 +1,21 @@
+from leapp.models import Model, fields
+from leapp.topics import SystemInfoTopic
+
+class OpenSshPermitRootLogin(Model):
+    topic = SystemInfoTopic
+
+    value = fields.StringEnum(['yes', 'prohibit-password', 'forced-commands-only', 'no'])
+    in_match = fields.Nullable(fields.List(fields.String()))
+
+
+class OpenSshConfig(Model):
+    ''' This model does not actually contain effective configuration options
```

Comment:
Malformed docstring - use double quotes, one-line summary (phrase ending in a period) + blank line + extended description for multi-line docstring, see the [guidelines](https://github.com/oamg/leapp-repository/blob/master/CONTRIBUTING.md#code-style).